service/awsconfig/v7/resource/cloudformation/create.go

```diff
@@ -42,7 +42,7 @@ func (r *Resource) ApplyCreateChange(ctx context.Context, obj, createChange inte
 			return microerror.Mask(err)
 		}
 
-		r.logger.LogCtx(ctx, "debug", "creating AWS cloudformation stack: created")
+		r.logger.LogCtx(ctx, "debug", "created the guest cluster main stack")
```

Comment:
FYI I needed to align this in the current active version in order to make the integration tests happy.